semcore/wizard/src/style/wizard.shadow.css

```diff
@@ -0,0 +1,81 @@
+@import '@semcore/utils/style/var.css';
+
+SWizard {
+  max-width: 980px;
+  max-height: 700px;
+  display: flex;
+}
+
+SSidebar {
+  background: var(--violet-600);
+  color: #fff;
+  font-weight: 700;
+  font-size: 16px;
+  padding: 40px 8px 34px;
+  width: 220px;
+  @media (max-width: 1060px) {
+    width: 44px;
+  }
+}
+
+SSidebarHeader {
+  margin: 0 20px 16px;
+  @media (max-width: 1060px) {
+    display: none;
+  }
+}
+
+SStepper {
+  display: flex;
+  font-size: 14px;
+  border-radius: 6px;
+  padding: 8px 12px;
+  cursor: pointer;
+  position: relative;
+  &:hover {
+    background: var(--violet-700);
+  }
+  &[active] {
+    background: var(--violet-500);
+  }
+  &:disabled,
+  &[disabled] {
+    opacity: 0.3;
+    cursor: default;
+  }
+}
+
+SStepNumber {
+  display: inline-block;
+  width: 20px;
```

Comment:
hm, if I want show this number `10000` ?

Reply:
discussed with Julia